docs/introduction.rst

```diff
@@ -4,7 +4,7 @@
 Introduction
 ============
 
-This is the documentation for tskit, the tree sequence toolkit. Tree sequences
+This is the documentation for tskit, the tree sequence toolkit. Succinct tree sequences
 are a highly efficient way of storing a set of related DNA sequences by encoding
```

Comment:
For grammar, shouldn't this read "The succinct tree sequence is a highly efficient way" or "Succinct tree sequences provide a highly efficient way"?

Reply:
True! Amended to "provide"